torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2371,6 +2380,49 @@ def extract_output_name(
             assert isinstance(output_name, list), type(output_name)
             output_args = output_name
 
+        # Special handling for torch.ops.higher_order.print
```

Comment:
can we create a dictionary like "custom_extern_kernel_codegen" that maps op to codegen implementation for cpp and python? this way we can keep all the code in one place. And then here we would just check if the op is in in the dictionary then we call that custom implementation

cc @BoyuanFeng